pdfreader/types/xref.py

```diff
@@ -41,7 +43,8 @@ class XRefEntry(BaseXrefEntry):
         No compressed objects support
     """
     def __init__(self, offset, number, generation, typ):
-        assert isinstance(offset, int) and offset >=0
```

Comment:
don't agree with this. assert statements are more pythonic and serve the same purpose. More readable, more compact, no performance drawback.

Reply:
Reverted this back to the original lines.  Thanks for the input!